client/src/components/ImageGallery.jsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import {Grid, Col, Row} from 'react-styled-flexboxgrid';
+
+const WrapperOuter = styled.div`
+  display: block;
+  box-sizing: border-box;
+`;
+
+const WrapperInner = styled.div`
+  align-items: stretch;
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: center;
+  box-sizing: border-box;
+  width: 1140px;
+  margin: auto;
+  position: relative;
+`;
+
+const Box1 = styled.div`
+  flex: 2;
+`;
+
+const Box23 = styled.div`
+  flex: 1;
+`;
+
+const Box45 = styled.div`
+  flex: 1;
+`;
+
+const Image1Container = styled.div`
+  flex: 1;
+`;
+
+const Image2Container = styled.div`
+  flex: 1;
+`;
+
+const Image3Container = styled.div`
+  flex: 1;
+`;
+
+const Image4Container = styled.div`
+  flex: 1;
+`;
+
+const Image5Container = styled.div`
+  flex: 1;
+`;
+
+const Image1 = styled.img`
+  cursor: pointer;
+  display: block;
+  height: 560px;
+  width: 560px;
+  object-fit: cover;
+  border-radius: 12px 0px 0px 12px;
+`
```

Comment:
On Line 61, you need a ';'.

Reply:
Also, it looks like the last style element in the below styled-components are missing a ';' as well. (ie line 68)

Reply:
got it. Done. (I need my linter working )